### ISSUE TITLE
Create interface for exploring and submitting observations using inaturalist API 

### ISSUE THREAD

**Issue by nkwsy** [nkwsy/Wildmile#137]
Use the [inaturalist api](https://api.inaturalist.org/v1/docs/) to pull data and sort info about observations on the Wild Mile and other sites.

inaturalist project id for 'Wild Mile Chicago' is `40796` 

Can Pull species counts via `https://api.inaturalist.org/v1/observations/species_counts?project_id=40796` and represent them in something similar to `image_card_grid.js` and display image, names, taxon and count info.

**Comment by DomAmato** [comment]
This might be better suited as a separate service that polls the inaturalist API and dumps it into the database otherwise you could potentially be hitting the API many times which will affect speed and have other potential issues like the domain being blacklisted. It already sort of happens with the species information since its pulling from many sites dozens of images. It could literally just be a cron job that runs once a day to refresh things.

**Comment by nkwsy** [comment]
I agree. could just have a collection for inaturalist queries an pull the info regularly then update the document.  I actually updated the documents for photos so that it stores and crops them all in an s3 bucket.
